과제3/index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="ko">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>디자인과제3</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap" rel="stylesheet">
+<link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="box">
+        <div class="header">
+            <img src="navigation.png" class="img-nav" alt="">
+            <p class="title">Machiatto</p>
+            <img src="cart.png" class="img-cart" alt="">
+        </div>
+            <button class="heart">
+                <img src="heart.png" width="24px" height="24px" alt="">
```

Comment:
다른 사진들과 사이즈가 같으므로 html에서 명시하지 않고 css에서 한 번에 명시하는 것이 좋아보입니다.
```suggestion
                <img src="heart.png" alt="">
```